src/renderer/app/components/SvgIcon/SvgIcon.vue

```diff
@@ -0,0 +1,29 @@
+<template>
```

Comment:
Needs a test that ensures that this component renders accordingly to all possible property combinations.